src/main/scala/com/fulcrumgenomics/fasta/CollectAlternateContigNames.scala

```diff
@@ -0,0 +1,175 @@
+import com.fulcrumgenomics.util.Io
+import enumeratum.EnumEntry
+
+import scala.collection.immutable.IndexedSeq
+
+/** Trait that entries in AssemblyReportColumn will extend. */
+sealed trait AssemblyReportColumn extends EnumEntry {
+  /** The column name in the assembly report. */
+  def key: String
+}
+
+/** Enum to represent columns in a NCBI assembly report. */
+object AssemblyReportColumn extends FgBioEnum[AssemblyReportColumn] {
+  def values: IndexedSeq[AssemblyReportColumn] = findValues
+
+  /** Allows the column to be build from the Enum name or the actual column name. */
+  override def apply(str: String): AssemblyReportColumn = {
+    values.find(_.key == str).getOrElse(super.apply(str))
+  }
+
+  case object SequenceName     extends AssemblyReportColumn { val key: String = "Sequence-Name" }
+  case object AssignedMolecule extends AssemblyReportColumn { val key: String = "Assigned-Molecule" }
+  case object GenBankAccession extends AssemblyReportColumn { val key: String = "GenBank-Accn" }
+  case object RefSeqAccession  extends AssemblyReportColumn { val key: String = "RefSeq-Accn" }
+  case object UcscName         extends AssemblyReportColumn { val key: String = "UCSC-style-name" }
+
+  val MissingValue: String   = "na"
+  val SequenceRole: String   = "Sequence-Role"
+  val SequenceLength: String = "Sequence-Length"
+}
+
+/** Trait that entries in SequencRole will extend. */
+sealed trait SequenceRole extends EnumEntry {
+  def key: String
+  def primary: Boolean
+}
+
+/** Enum to represent the various types of sequence-roles in NCBI assembly report. */
+object SequenceRole extends FgBioEnum[SequenceRole] {
+  def values: IndexedSeq[SequenceRole] = findValues
+
+  /** Allows the column to be build from the Enum name or the actual sequence-role column name. */
+  override def apply(str: String): SequenceRole = {
+    values.find(_.key == str).getOrElse(super.apply(str))
+  }
+
+  case object AltScaffold         extends SequenceRole { val key: String = "alt-scaffold"; val primary: Boolean = false }
+  case object AssembledMolecule   extends SequenceRole { val key: String = "assembled-molecule"; val primary: Boolean = true }
+  case object FixPatch            extends SequenceRole { val key: String = "fix-patch"; val primary: Boolean = false }
+  case object NovelPatch          extends SequenceRole { val key: String = "novel-patch"; val primary: Boolean = false }
+  case object UnlocalizedScaffold extends SequenceRole { val key: String = "unlocalized-scaffold"; val primary: Boolean = false }
+  case object UnplacedScaffold    extends SequenceRole { val key: String = "unplaced-scaffold"; val primary: Boolean = false }
+}
+
+
+@clp(description =
+  """
+    |Collates the alternate contig names from an NCBI assembly report.
+    |
+    |The input is be the `*.assembly_report.txt` obtained from NCBI.
+    |
+    |The output will have the first column is the primary name and the second column is an alternative name.  If there
```

Comment:
"The output will have primary name as the first column and alternate name as the second column." Reads a little better to me